Shields.DataStructures.Async/AsyncBoundedQueue.cs

```diff
@@ -299,40 +292,38 @@ public Task EnqueueAsync(T value, CancellationToken cancellationToken)
         }
 
         /// <summary>
-        /// Returns a disposable that completes all waiting dequeue operations with the specified value.
+        /// Completes all waiting dequeue operations with the specified value.
         /// </summary>
         /// <param name="value">The value to return from all waiting dequeue operations.</param>
-        /// <returns>The disposable that completes all waiting dequeue operations.</returns>
-        public IDisposable CompleteAllDequeue(T value)
+        public void CompleteAllDequeue(T value)
         {
-            return dequeueQueue.DequeueAll(value);
+            dequeueQueue.DequeueAll(value);
         }
 
         /// <summary>
-        /// Returns a disposable that cancels all waiting dequeue operations.
+        /// Cancels all waiting dequeue operations.
         /// </summary>
-        /// <returns>The disposable that cancels all waiting dequeue operations.</returns>
-        public IDisposable CancelAllDequeue()
+        /// <param name="cancellationToken"></param>
+        public void CancelAllDequeue(CancellationToken cancellationToken)
         {
-            return dequeueQueue.CancelAll();
+            dequeueQueue.CancelAll(cancellationToken);
         }
 
         /// <summary>
-        /// Returns a disposable that completes all waiting enqueue operations.
+        /// Completes all waiting enqueue operations.
         /// </summary>
-        /// <returns>The disposable that completes all waiting enqueue operations.</returns>
-        public IDisposable CompleteAllEnqueue()
+        public void CompleteAllEnqueue()
         {
-            return enqueueQueue.DequeueAll();
+            enqueueQueue.DequeueAll();
         }
 
         /// <summary>
-        /// Returns a disposable that cancels all waiting enqueue operations.
+        /// Cancels all waiting enqueue operations.
         /// </summary>
-        /// <returns>The disposable that cancels all waiting enqueue operations.</returns>
-        public IDisposable CancelAllEnqueue()
+        /// <param name="cancellationToken"></param>
```

Comment:
Please change the parameter comment to the following.

    /// <param name="cancellationToken">The cancellation token.</param>